db/querys.js

```diff
@@ -1,11 +1,10 @@
 const mysql = require('mysql');
 
 const connection = mysql.createConnection({
-  host: '70.121.184.146',
+  host: 'localhost',
   user: 'root',
   password: 'duckDuckGoose',
-  database: 'bestbuy_db',
-  port: 3001
+  database: 'bestbuy_db'
 });
 
```

Comment:
Not an issue, But for future reference when you deploy on EBS: Use this article to Outline what you need in the connection:
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create-deploy-nodejs.rds.html